01.md

```diff
@@ -139,7 +139,7 @@ The `ids`, `authors`, `#e` and `#p` filter lists MUST contain exact 64-character
 
 The `since` and `until` properties can be used to specify the time range of events returned in the subscription. If a filter includes the `since` property, events with `created_at` greater than or equal to `since` are considered to match the filter. The `until` property is similar except that `created_at` must be less than or equal to `until`. In short, an event matches a filter if `since <= created_at <= until` holds.
 
-All conditions of a filter that are specified must match for an event for it to pass the filter, i.e., multiple conditions are interpreted as `&&` conditions.
+All different fields of a filter, if present, must match for an event for it to pass the filter. For example, if a filter has `{"kinds": [1, 3], "authors": [<pubkey>]}` it'll be interpreted as `(kind === 1 || kind === 3) && author === <pubkey>`.
```

Comment:
```suggestion
All different fields of a filter, if present, must match for an event for it to pass the filter. For example, if a filter has `{"kinds": [1, 3], "authors": [<pubkey>]}` it'll be interpreted as `(kind == 1 || kind == 3) && author == <pubkey>`.
```

I think the equality operator is `==` in many languages.

Reply:
Sorry, I mostly Deno. For me it doesn't matter, committing your suggestion.